ubiquibot-config-default.json

```diff
@@ -8,6 +8,8 @@
   "disable-analytics": false,
   "comment-incentives": false,
   "register-wallet-with-verification": false,
+  "openai-api-key": "",
```

Comment:
If this is not set or is an empty string then the OpenAI API will throw an error by default anyway

Reply:
- https://github.com/Keyrxng/UbiquityBotTesting/issues/44#issuecomment-1718081294

Reply:
Can you change the error to be like? 

```diff
! start with a bang
```

> If this is not set or is an empty string then the OpenAI API will throw an error by default anyway, should I still implement null and throw our own error?

It would provide clarity to our early users if we explain how to enable this feature. Basically it should say something like

"You must configure the `openai-api-key` property in the bot configuration in order to use AI powered features."

Reply:
- #770

Reply:
- https://github.com/Keyrxng/UbiquityBotTesting/pull/46#issuecomment-1719861580